jp2_online/templates/estudios_socioeconomicos/focus_mode.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'layouts/dashboard_base.html' %}
+
+{% block content %}
+    <div class="row">
+      <div class="col-md-12 col-sm-12 col-xs-12">
+        <div class="x_panel">
+          
+          <div class="x_title">
+            <h2>{{ estudio }}<small>Condensado Familia</small></h2>
+
+            <div class="clearfix"></div>
+          </div>
+          <div class="x_content">
+            
+            <div class="col-md-12 col-sm-12 col-xs-12 profile_left">
+              <div class="profile_img">
+                
+              </div>
+              <h3>Familia {{ estudio }}</h3>
+
+              <!-- Verificar el status del estudio -->        
+              <a class="btn btn-success pull-right">
+                <i class="fa fa-edit m-right-xs"></i>
+                Editar Estudio
+              </a>
+
+              <ul class="list-unstyled user_data">
+                <li>
+                    Localidad {{ estudio.familia.localidad }}
+                </li>
+
+                <li>
+                    Capturista:
+                    {{ estudio.capturista.user.first_name }}
+                    {{ estudio.capturista.user.last_name }}
+                </li>
+                
+                <li>
+                    Status estudio: {{ estudio.status }}
```

Comment:
you should change this to: estudio.get_status_display
check [this](https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.get_FOO_display)